src/wallet/wallet.cpp

```diff
@@ -4719,40 +4407,6 @@ bool CWallet::CreateZerocoinSpendModelV2(
     return true;
  }
 
-// TODO(martun): check this function. These string denominations which come from
-// outside may not be parsed properly.
-bool CWallet::CreateSigmaSpendModel(
-        CWalletTx& wtx,
-        string &stringError,
-        string& thirdPartyAddress,
-        const vector<string>& denomAmounts,
-        bool forceUsed) {
-    if (!fFileBacked)
-        return false;
-
-    vector<sigma::CoinDenomination> denominations;
-    for(vector<string>::const_iterator it = denomAmounts.begin(); it != denomAmounts.end(); it++){
-        const string& denomAmount = *it;
-        sigma::CoinDenomination denomination;
-        // Amount
-        if (!StringToDenomination(denomAmount, denomination)) {
-            stringError = "Unable to convert denomination.";
-            return false;
-        }
-        denominations.push_back(denomination);
-    }
-    vector<Scalar> coinSerials;
-    uint256 txHash;
-    vector<GroupElement> zcSelectedValues;
-    stringError = SpendMultipleSigma(
-        thirdPartyAddress, denominations, wtx,
-        coinSerials, txHash, zcSelectedValues, forceUsed);
-    if (stringError != "")
-        return false;
-    return true;
-}
-
-
 /**
  * @brief CWallet::CreateZerocoinMintTransaction
  * @param vecSend
```

Comment:
This function looks like it could benefit from cleanup too. Lots of commented out code. And probably needs to be renamed